### ISSUE TITLE
Types of Testing: Diagram or Tag Cloud?

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/practical-full-stack-testing#26]
What communicates the various types of testing better?

![manual testing suited to](http://i.imgur.com/lCPdBav.jpg)

**Comment by iteles** [comment]
I find tag clouds become unclear very quickly. The right diagram means keeping it clean.

**Comment by nelsonic** [comment]
Yeah, the tag cloud has pretty much been relegated as a UI element due to its useability...
Will work on a diagram tomorrow morning. :+1:
